cdp-langchain/examples/chatbot/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@coinbase/cdp-langchain-chatbot-example",
   "description": "CDP Agentkit Node.js SDK Chatbot Example",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
we are not publishing the chatbot project right now so we don't need to bump the version here.

Reply:
roger!